src/api/app/assets/javascripts/webui/user_profile.js

```diff
@@ -0,0 +1,7 @@
+function placeInvolvement() { // jshint ignore:line
+  var parent = $('#involvement-and-activity');
```

Comment:
`parent` is the container of `#involvement`, right? If so, I would name this variable `container`.